src/bundle/Controller/SectionController.php

```diff
@@ -179,8 +178,13 @@ public function listAction(Request $request): Response
             $assignableSections[$section->id] = $this->canUserAssignSectionToSomeContent($section);
         }
 
+        // User can add Section only if he has access to view without any limitation because he must see newly created Section.
+        $canView = $this->permissionResolver->hasAccess('section', 'view') === true;
+        $canEdit = $this->isGranted(new Attribute('section', 'edit'));
```

Comment:
Why is that you are using `$this->isGranted(...)` and `$this->permissionResolver->hasAccess(...)`? Isn't `isGranted` method just a wrapper for `PermissionResolver::hasAccess` internally?